packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,265 @@
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+<Card backgroundLevel={1}>
+  <Card.Header 
+    title="Payments Links" 
+    subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+    titlePrefix={<DollarIcon size="xl" />}
+    titleSuffix={<Counter amount={20} />}
+    trailingVisual={<Badge variant="positive">NEW</Badge>}
+  />
+  <Card.Body>
+    Card Body Content
+  </Card.Body>
+  <Card.Footer
+    title="Card Footer Title" 
+    subtitle="Card footer subtitle" 
+    actions={{
+      primaryAction: { text: 'Know more', onClick: () => {} },
+      secondaryAction: { text: 'Read Docs', onClick: () => {} },
+    }} 
+  />
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop              | Type   | Default | Description                                                                            | Required |
+| ----------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `backgroundLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
+
+
+### `Card.Header` API
+
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `title`          | `string`          | `undefined` | Title of the Card                                        | ✅        |
+| `subtitle`       | `string`          | `undefined` | Subtitle of the Card                                     |          |
+| `titlePrefix`    | `React.ReactNode` | `undefined` | Prefix element placed before title text (eg: Icon)       |          |
+| `titleSuffix`    | `React.ReactNode` | `undefined` | Suffix element placed after title text (eg: Counter)     |          |
+| `trailingVisual` | `React.ReactNode` | `undefined` | Trailing visual element placed on right side of the card |          |
+
+### `Card.Footer` API
+
+| Prop       | Type                                     | Default     | Description                                | Required |
+| ---------- | ---------------------------------------- | ----------- | ------------------------------------------ | -------- |
+| `title`    | `string`                                 | `undefined` | Title of the Card                          | ✅        |
+| `subtitle` | `string`                                 | `undefined` | Subtitle of the Card                       |          |
+| `actions`  | `{ primary: Action, secondary: Action }` | `undefined` | Renders a primary/secondary action buttons |          |
+
+```ts
+type Action = {
+  onClick: () => void;
+  text: string;
```

Comment:
We should expose the following props too:

1. `variant` (Unsure about this but there is a use-case for tertiary button in the design)
2. `type` ( Could be needed for reset or submit buttons in a card)
3.  `accessibilityLabel`
4. `isLoading`
5. `isDisabled`
6. `icon` & `iconPosition`?

Reply:
🤔 We can just expose variant, isLoading, type for now? 
Because not sure if we will even have a icon only button usecase in the footer actions.

Reply:
Done!